livre/eq/Config.cpp

```diff
@@ -194,16 +199,22 @@ bool Config::init( const int argc, char** argv )
 
 uint32_t Config::frame()
 {
+    if( _impl->dataFrameRange == INVALID_FRAME_RANGE )
+        return eq::Config::finishFrame();
```

Comment:
Because, I want to return when I dont have a legal frame range and if you check the end of the function it is returning with this function. I am not aware how Eq will be effected by that, and I didnt have a warning in the code review earlier taht I have believed it is legal so I've pushed it.